src/css/brandColors.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable n/no-sync */
+/* eslint-disable jsdoc/require-jsdoc */
+import path from 'path';
+
+import { getCssColorVariables } from '../../utils';
+import jsonData from '../figma/brandColors.json';
+
+describe('CSS Variables', () => {
+  const cssFilePath = path.join(__dirname, 'generated-brand-colors.css');
+  const cssVariables = getCssColorVariables(cssFilePath, 'brand-colors');
+
+  it('should match JSON values', () => {
+    for (const [colorGroup, shades] of Object.entries(jsonData)) {
```

Comment:
Updating to get brand colors from current token json
```suggestion
    for (const [colorGroup, shades] of Object.entries(
      jsonData.global.brandColors,
    )) {
```